### ISSUE TITLE
Wrong units for SLP and TROPP in preprocessed GEOS-IT ExtData.rc

### ISSUE THREAD

**Issue by yuanjianz** [geoschem/gchp#398]
### Name and Institution (Required)

Name:Yuanjian Zhang
Institution: Washington University in St. Louis

### Confirm you have reviewed the following documentation

- [x] [Support guidelines](https://gchp.readthedocs.io/en/latest/reference/SUPPORT.html)
- [x] [User manuals](https://gchp.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [x] [Debugging GEOS-Chem and HEMCO errors](https://gchp.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x] [Current and past Github issues](https://github.com/geoschem/GCHP/issues)

### Description of your issue or question

In GEOS-IT preprocessed files, all pressure terms (PS, SLP and TROPP) are scaled to hPa. However, Extdata.rc only correctly labels PS as hPa, while SLP and TROPP are still in Pa with a scaling factor of 0.01. The incorrect TROPP might affect chemistry and transport tracer benchmarking.
```
SLP     Pa  N Y F0;003000 none 0.01 SLP      ./MetDir/%y4/%m2/GEOSIT.%y4%m2%d2.A1.05x0625.nc
TROPP   Pa  N Y F0;003000 none 0.01 TROPPT   ./MetDir/%y4/%m2/GEOSIT.%y4%m2%d2.A1.05x0625.nc
```
@lizziel

**Comment by lizziel** [comment]
Thanks for reporting this! We will put a fix into 14.3.1. See PR https://github.com/geoschem/geos-chem/pull/2207.

**Comment by yantosca** [comment]
We can now close this issue as PR #2207 has now been merged into the "no-diff-to-benchmark" development stream.
